### DISCUSSION TITLE
ghippo/best-practice/oem/oem-in

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#5233] [Announcements]
# ghippo/best-practice/oem/oem-in

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/best-practice/oem/oem-in.html

**Comment by panpan0000** [comment]
gproduct-demo-main.tar.gz 文件在git clone 这个仓库 https://gitlab.daocloud.cn/henry.liu/gproduct-demo

**Comment by windsonsea** [comment]
Fixed in #5265

**Reply by panpan0000** [reply]
既然例子用了lable-studio，为什么不用label-studio作为OIDC的client端，来示范对接远端的demo-dev的SSO server端？反而搬出第二套DCE。 这个文档无法往其他类似label-studio的项目推动

**Reply by lemontrees123** [reply]
lable-studio不支持用户体系打通，不能完整的演示整个过程，因此用户体系打通步骤采用了两套DCE5相互对接的演示方式。另外，由于无法真实的在本地部署客户环境并完整演示上述全部流程，因此请独立看待上述每一个步骤，各步骤相互独立互不影响。
